includes/wikia/tasks/Tasks/ImageReviewTask.class.php

```diff
@@ -108,6 +108,68 @@ public function deleteFromQueue( Array $aDeletionList ) {
 		}
 	}
 
+	public function addToQueue() {
+		global $wgExternalDatawareDB;
+		$title = $this->getTitle();
+		$wikiId = $this->getWikiId();
+		$latestRevId = $title->getLatestRevID( \Title::GAID_FOR_UPDATE );
+
+		$imageData = [
+			'wiki_id' => $wikiId,
+			'page_id' => $title->getArticleID(),
+			'revision_id' => $latestRevId,
+			'user_id' => wfLocalFile( $title )->getUser( 'id' ),
+			'last_edited' => \Revision::newFromId( $latestRevId, \Revision::READ_LATEST )->getTimestamp(),
+			'top_200' => $this->isTop200( $wikiId ) ? 1 : 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "top_200" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)